nixos/modules/services/x11/desktop-managers/gnome3.nix

```diff
@@ -68,6 +68,38 @@ in
       core-shell.enable = mkEnableOption "GNOME Shell services";
       core-utilities.enable = mkEnableOption "GNOME core utilities";
       games.enable = mkEnableOption "GNOME games";
+
+      experimental-features = {
```

Comment:
Style nit: why kebab case rather than camel case? Just mimicking `nix`'s `experimental-features` flag/configuration option?

(Doesn't really matter, but it seems we use camel case everywhere else.)

Reply:
The options above mimic the element names for GNOME OS from gnome-build-meta. The actual dconf key this setting operates on is `/org/gnome/mutter/experimental-features`, and they just throw whatever  feature in there. In the future it would be nice to have https://github.com/NixOS/nixpkgs/issues/54150 to extend with more experimental-feature options. (though I do see how it's named similar to how nix has experimental-features). I just happened to group it in services.gnome3 because that namespace made the most sense.

Though, you're correct we mix casing for options. But for these ones it would be really ugly to read in camel.

Reply:
Ah, OK, fair enough. Carry on :)